iso20022-core/src/main/java/com/prowidesoftware/swift/model/mx/SupplementaryDataUtils.java

```diff
@@ -0,0 +1,71 @@
+package com.prowidesoftware.swift.model.mx;
+
+import com.prowidesoftware.swift.utils.SafeXmlUtils;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SupplementaryDataUtils {
```

Comment:
Agregale javadoc aca y en el metodo publico epxlicando para que sirve y con el since.
Copia también para aca el test de integrator, sácale cualquier dato sensitivo a los samples

Reply:
Ok